src/lib/progress.c

```diff
@@ -31,6 +32,9 @@ static progress_fn_t progress_function = NULL;
 static start_fn_t start_function = NULL;
 static end_fn_t end_function = NULL;
 
+static char spinner[5] = { '|', '/', '-', '\\' };
+static time_t begin = 0;
```

Comment:
Can't begin be a static inside the spinner function, like index?

Reply:
I am not sure why did I get an error the first time I tried, but works now :confused: